README.md

```diff
@@ -152,9 +152,9 @@ If you find a bug that disrupts you, please file an issue with its impact to you
 | MUL | ✅ |✅ |
 | SUB | ✅ |✅ |
 | DIV | ✅ |✅ |
-| SDIV | ✅ |❓ |
-| MOD | ✅ |❓ |
-| SMOD | ✅ |❓ |
+| SDIV | ✅ |✅ |
+| MOD | ✅ |✅ |
```

Comment:
this is a false positive. We don't have test coverage for `MOD`. But this is fine.